### PR TITLE
Update binding-types-reference.md

### PR DESCRIPTION
**Pull request by nielscup**

### CODE REVIEW

**Comment by PRMerger7** [comment]
@nielscup : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by davidbritch** [comment]
@chamons Can you advise if this PR is correct?

File: docs/cross-platform/macios/binding/binding-types-reference.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ interface MyType : [Protocol1, Protocol2] {
 ```
 
 Every interface in your contract definition that has the 
-[`[BaseType]`](#BaseTypeAttribute) attribute that declares the base type for 
+[`[BaseType]`](#BaseTypeAttribute) attribute declares the base type for 
 the generated object. In the above declaration a `MyType` class C# type will 
-be generated that binds to an Objective-C type called `MyType`.
+be generated that binds to an Objective-C type called `BTYPE`.
```
Comment (chamons) [inline docs/cross-platform/macios/binding/binding-types-reference.md:36]: This is incorrect.

Comment (spouliot) [inline docs/cross-platform/macios/binding/binding-types-reference.md]: no, the example is binding `MyType`

**Comment by chamons** [comment]
The first grammar fix, and the last part are fine. I've marked the one part that is incorrect.

**Comment by chamons** [comment]
First grammar and last bit are fine. @spouliot marked the incorrect bit.

**Comment by PRMerger18** [comment]
@davidbritch : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by davidbritch — APPROVED** [review]
Thanks @nielscup

**Pull request merged**